3-12-2018/grzesiek/weather/lib/weather_fetcher.rb

```diff
@@ -0,0 +1,42 @@
+require 'http'
+require 'byebug'
+require 'csv'
+
+module WeatherFetcher
+  module_function
+
+  API_KEY = '5dELbO869g0TMpA6mBrQdQ8zZstnKRPh'
+
+  def sanitize_location(location_to_sanitize)
+    location_to_sanitize.empty? && raise(ArgumentError)
+
+    sanitized_string = location_to_sanitize.tr("ąęźżóńł ", "aezzonl-")
+  end
+
+  def get_location_key(location)
+    response = HTTP.get("http://dataservice.accuweather.com/locations/v1/cities/search?apikey=#{API_KEY}&q=#{location}&details=true")
```

Comment:
'http://dataservice.accuweather.com/' could be separate HOST constant

Reply:
Good idea, thanks.